deeplearning4j/deeplearning4j-modelexport-solr/src/main/java/org/deeplearning4j/nn/modelexport/solr/ltr/model/ScoringModel.java

```diff
@@ -106,24 +106,28 @@ public float score(float[] modelFeatureValuesNormalized) {
     return outputScore(model, modelFeatureValuesNormalized);
   }
 
+  public static float outputScore(Model model, float[] modelFeatureValuesNormalized) {
+    final INDArray input = Nd4j.create(modelFeatureValuesNormalized);
+    final INDArray output = output(model, input);
+    return output.getFloat(0);
+  }
+
   /**
    * Currently supports {@link MultiLayerNetwork} and {@link ComputationGraph} models.
    * Pull requests to support additional <code>org.deeplearning4j</code> models are welcome.
    */
-  public static float outputScore(Model model, float[] modelFeatureValuesNormalized) {
-
-    final INDArray input = Nd4j.create(modelFeatureValuesNormalized);
+  public static INDArray output(Model model, INDArray input) {
```

Comment:
The changes here to `ScoringModel.java` are unrelated to this pull request and are intended to 'go away' via the #5428 refactor.